source/docs/magento1/exposing-additional-attributes.md

```diff
@@ -0,0 +1,106 @@
+---
+id: exposing-additional-attributes
+title: Exposing additional attributes
+---
+
+In case where you need to expose new attributes on the existing endpoint you can simply add this attribute on your `api2.xml` file. For that, you need to retrieve resource name (API_UNIQUE_NAME value if you follow the basic structure shown in "Add custom endpoint section") and add your attribute(s) into the `attributes` node.
+
+- How to retrieve the resource name: you can retrieve this data on all `api2.xml` file.
+- Expose attribute `custom_attribute` for customer data:
+```
+# Your api2.xml file
+...
+ <resources>
+    <customer>
+        <attributes>
+            <custom_attribute />
+        </attributes>
+    </customers>
+</resources>
+...
+```
+- Expose attribute `custom_attribute` for product data:
+```
+# Your api2.xml file
+...
+ <resources>
+    <product>
+        <attributes>
+            <custom_attribute />
+        </attributes>
+    </customers>
+</resources>
+...
+```
+- Expose attribute `custom_attribute` for category data:
+```
+# Your api2.xml file
+...
+ <resources>
+    <category>
+        <attributes>
+            <custom_attribute />
+        </attributes>
+    </customers>
+</resources>
+...
+```
+- Forcing attributes:
+Sometime some attributes is forcing exclude. You can retrieve an example on `app/code/community/Clockworkgeek/Extrarestful/etc/api2.xml` file:
```

Comment:
I don't understand what you mean here. Is it about excluding an attribute or allowing to retrieve an attribute?